src/palindrom-network-channel.js

```diff
@@ -1,5 +1,6 @@
 import URL from './URLShim';
 import { PalindromError, PalindromConnectionError } from './palindrom-errors';
+import { Heartbeat, NoHeartbeat } from './heartbeat';
```

Comment:
All changes in this file seem unintended for this PR, because they are not explained in the PR description nor in the commit message